build/deploy-env/conda-aarch64/Dockerfile

```diff
@@ -45,8 +45,6 @@ USER saluser
 WORKDIR ${WORKDIR}
 
 ENV numpy=1.20
-ENV ts_idl=2.3.0
```

Comment:
You need both ts_idl and ts_salobj (see line 59 and 60), but they must be `ARG` not `ENV`. You probably need to convert `ENV numpy=1.20` to `ARG numpy=1.20` as well.

Reply:
Good catch. I totally overlooked that. Done.